Firestore/Example/Tests/SpecTests/FSTSyncEngineTestDriver.mm

```diff
@@ -247,6 +252,38 @@ - (void)drainQueue {
   return _currentUser;
 }
 
+//- (const std::vector<std::shared_ptr<EventListener<Empty>>>&)snapshotsInSyncListeners {
+//  return _snapshotsInSyncListeners;
+//}
+
+- (void)incrementSnapshotsInSyncEvents {
+  _snapshotsInSyncEvents += 1;
+}
+
+- (void)resetSnapshotsInSyncEvents {
+  _snapshotsInSyncEvents = 0;
+}
+
+- (void)addSnapshotsInSyncListener {
+  std::shared_ptr<EventListener<Empty>> eventListener = EventListener<Empty>::Create(
+      [self](const StatusOr<Empty> &v) { [self incrementSnapshotsInSyncEvents]; });
```

Comment:
Since `v` is unused, don't name the argument.

Reply:
You've changed the type from `const StatusOr<Empty> &` to `const StatusOr<Empty>`. Instead of merely ignoring the argument, this is now copying and discarding the argument. Please only remove the `v` without altering the type.